pypesto/objective/priors.py

```diff
@@ -0,0 +1,345 @@
+import numpy as np
+from typing import Callable, Dict, List, Sequence, Tuple, Union
+from copy import deepcopy
+
+from .function import ObjectiveBase
+from .aggregated import AggregatedObjective
+from .constants import MODE_FUN, MODE_RES, FVAL, GRAD, HESS
+
+ResultDict = Dict[str, Union[float, np.ndarray, Dict]]
+
+
+class NegLogPriors(AggregatedObjective):
+    """
+    Aggregates different forms of negative log-prior distributions.
+
+    Allows to distinguish priors from the likelihood by testing the type of
+    an objective.
+
+    Consists basically of a list of individual negative log-priors,
+    given in self.objectives.
+    """
+
+
+class NegativeLogParameterPriors(ObjectiveBase):
```

Comment:
use abbreviations consistently, i.e. here also `NegLogParameterPriors` similar to above